scripts/rust-analyze.sh

```diff
@@ -54,6 +56,8 @@ if [ -d "$RUST_ANALYSIS_IN" ]; then
   fi
 
   $MOZSEARCH_PATH/tools/target/release/rust-indexer \
+    "$CONFIG_FILE" \
+    "$TREE_NAME" \
```

Comment:
rust-indexer hasn't changed, so it doesn't expect those arguments.
```suggestion
```

Reply:
Good eye!  I had a patch later in my stack with the comment "Fold this: Undo erroneous additional args to rust-analyze.sh" that I missed!  (I'm pretty sure this was a situation where I made the changes in the wrong editor buffer and only noticed it later when I went to run the searchfox repo tests.)